src/main/java/com/atlassian/bitbucket/jenkins/internal/http/HttpRequestExecutorImpl.java

```diff
@@ -30,20 +30,25 @@ public HttpRequestExecutorImpl(Call.Factory httpCallFactory) {
     @Override
     public <T> T executeGet(HttpUrl url, BitbucketCredentials credential, ResponseConsumer<T> consumer) {
         Request.Builder requestBuilder = new Request.Builder().url(url);
-        if (credential != ANONYMOUS_CREDENTIALS) {
-            requestBuilder.addHeader(AUTHORIZATION_HEADER_KEY, credential.toHeaderValue());
-        }
+        addAuthorization(credential, requestBuilder);
```

Comment:
add**Authentication** we can't authorise, we can only add authentication details